aptos-move/framework/src/natives/string_utils.rs

```diff
@@ -285,6 +285,9 @@ fn native_format_impl(
             )?;
             out.push('}');
         },
+        MoveTypeLayout::Tagged(tag, ty) => match tag {
```

Comment:
String utils are not correct with aggregators actually, because they access Move value of aggregator struct directly. I think it is a good idea to print something which would say it is maybe a correct value👍